com.exok.Celeste64.yml

```diff
@@ -0,0 +1,46 @@
+id: com.exok.Celeste64
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet8
+build-options:
+  append-path: "/usr/lib/sdk/dotnet8/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet8/lib"
+  env:
+    PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet8/lib/pkgconfig"
+command: /app/bin/celeste64/bin/Release/net8.0/linux-x64/Celeste64
```

Comment:
comman should just be the name of the comand that in the the PATH. NOT an full path.

Reply:
also how would that work on aarch64?